scanner/matcher/updater/vuln/updater.go

```diff
@@ -483,7 +488,7 @@ func (u *Updater) runMultiBundleUpdate(ctx context.Context) error {
 		zlog.Info(ctx).Msg("starting bundle update")
 		if err := u.updateBundle(ctx, bundleF, zipTime, prevTime); err != nil {
 			zlog.Error(ctx).Err(err).Msg("updating bundle failed")
-			return fmt.Errorf("updating bundle %s: %w", bundleF.Name, err)
+			return false, fmt.Errorf("updating bundle %s: %w", bundleF.Name, err)
```

Comment:
Should this bool reflect if any file was processed successful? ie: if 5 files were loaded successful and #6 fails, should we run GC because 5 files were loaded?

Reply:
That's a good idea. I also checked what Claircore does, and they always run the GC, if upon failure. We don't have to always run it (point of this PR), but we can still run it with partial success like you suggest. I'll add that.

Claircore ref: https://github.com/quay/claircore/blob/v1.5.31/libvuln/updates/manager.go#L233

You'll find the errors are sent to a channel and only read after the GC runs

Reply:
Thinking about it again, and we will just try again in a few more minutes for the 6th to (hopefully) complete successfully. We also run a full GC process every 24 hours. I'm tempted to actually keep things as-is for now to keep it simple, but I'm ok with creating a ticket for this. What do you think?

Reply:
Agree with keeping things simple, wasn't sure the implications when made the suggestion - given the scheduled re-run and 'no harm done' - seems OK to me as is

Reply:
I created a Jira for this, so I'll resolve this comment